roads.mss

```diff
@@ -2238,10 +1765,13 @@ come in as well.
     line-color: @track-fill;
   }
   [type='railway'] {
-    line-color: @rail-line;
-    line-dasharray: 1,1;
-    [type='subway'] { line-opacity: 0.67; }
-    [zoom>15] { line-dasharray: 1,2; }
+    line-width: 0;
```

Comment:
This can probably be skipped provided that `[type='railway']` is written `[type='railway'][tunnel=0]` (previous and next lines fused together).